src/app/modules/shared-components/components/collapsible-section/collapsible-section.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, ContentChild, Input, TemplateRef } from '@angular/core';
```

Comment:
Very clean solution :+1:
now progress-~section~ and boolean-~section~ can be used with any collapse widget, even without any widget :+1: :+1: